scripts/index.js

```diff
@@ -1,10 +1,10 @@
-const buttonClipBoardTemplate = `<button class='button-copy-clipboard btn btn-light btn-sm' onclick="copyToClipBoard(this.parentElement.textContent)">Copier</button>`
+const buttonClipBoardTemplate = `<button class='button-copy-clipboard btn btn-light btn-sm' onclick="copyToClipBoard(this)">Copier</button>`
 
 document.addEventListener('DOMContentLoaded', () => {
 
-    const elements = document.getElementsByClassName('language-ruby');
+    const elements = document.getElementsByClassName('language-ruby highlighter-rouge');
```

Comment:
du coup, le bouton copier apparait que pour le code ruby: 

Tu peux peut-être utiliser:

```suggestion
    const elements = document.getElementsByClassName('highlighter-rouge');
```

Reply:
oui en effet du coup j'avais fait ça au moment du "copier" pour être sur de ne prendre que le contenu du block code : 
`const text = e.parentElement.getElementsByTagName('code')[0].innerText;`
il me semble qu'avec juste `language-ruby highlighter-rouge` il prenait quand même le "copier" du bouton je vais re-tester.

Reply:
done!